### PR TITLE
Feat/onboarding redirect to project

### PR DESCRIPTION
**Pull request by FranzPoize**

### CODE REVIEW

File: src/components/SignupForm/index.js [hunk 1]
```diff
@@ -24,8 +24,8 @@ class SignupForm extends Component {
 	}
 
 	render() {
-		const {shouldRedirect} = this.state;
-		const from = this.props.from || '/app/onboarding';
+		const {shouldRedirect, projectId} = this.state;
+		const from = this.props.from || `/app/onboarding/${projectId}`;
```
Comment (YoruNoHikage) [inline src/components/SignupForm/index.js]:
```suggestion
		const from = this.props.from || `/app/onboarding?projectId=${projectId}`;
```

File: src/containers/App/Onboarding/index.js [hunk 2]
```diff
@@ -99,14 +99,15 @@ class Onboarding extends Component {
...
-			return <Redirect to="/app" />;
+			return <Redirect to={`/app/projects/${projectId}/see`} />;
 		default:
 			return false;
 		}
 	};
 
 	render() {
 		const {step} = this.state;
+		const {projectId} = this.props.match.params;
```
Comment (YoruNoHikage) [inline src/containers/App/Onboarding/index.js]:
```suggestion
		const queryString = new URLSearchParams(this.props.location.search); 
		const projectId = queryString.get('projectId');`
```

File: src/containers/App/index.js [hunk 3]
```diff
@@ -78,7 +78,10 @@ class App extends Component {
 									isAllowed={data && data.me}
 								/>
 								<ProtectedRoute
-									path="/app/onboarding"
+									path={[
+										'/app/onboarding/:projectId',
+										'/app/onboarding',
+									]}
```
Comment (YoruNoHikage) [inline src/containers/App/index.js]: No need for this

**Review by YoruNoHikage — CHANGES_REQUESTED** [review]
Let's not make another route

**Comment by yannickmathey** [comment]
I fixed a typo on 'bienvenuE' -> c490d590a564474d8c5a77ba7eb495ca9d41512f

**Review by YoruNoHikage — APPROVED** [review]

**Pull request merged**